src/Sentry/SentryOptions.cs

```diff
@@ -77,6 +77,16 @@ public bool IsGlobalModeEnabled
         set => _isGlobalModeEnabled = value;
     }
 #endif
+    private NdkHandlerStrategy _handlerStrategy = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
+    /// <summary>
+    /// Handler Strategy
```

Comment:
Please describe further what this is for.
Consider adding `<seealso href="https://github.com/dotnet/android/issues/9055#issuecomment-2261347912">`